python_modules/libraries/dagster-elt/dagster_elt/ops.py

```diff
@@ -0,0 +1,28 @@
+from dagster import Config, op, In, Out, Nothing, Output
+
+from typing import Any, Iterable, Generator, List
+
+from .types import SlingOutput
+from .resources import SlingResource, SlingSource, SlingTarget, SlingMode
+
+
+class SlingSyncConfig(Config):
+    source: SlingSource
+    target: SlingTarget
+    mode: SlingMode
+
+
+@op(
+    ins={"start_after": In(Nothing)},
+    out=Out(
+        List[str],
+        description="TODO",
+    ),
+    tags={"kind": "sling"},
+)
+def sling_sync_op(context, config: SlingSyncConfig, sling: SlingResource) -> List[str]:
```

Comment:
Do we feel like it's important to include this? I'd lean towards being opinionated the assets are the right way to do this and not widening the surface area to maintain.

Reply:
Happy to exclude this, wasn't clear to me that it added anything.

Reply:
+1